blog/exporters.md

```diff
@@ -176,18 +234,18 @@ In the `secadvisor.yml` file, uncomment the `iam_endpoint` field and set it to `
 
 For running tests on a local machine, it is possible to set up a local Minio object store.
 
-For details, see the instructions found at [Secadvisor Developer's README](https://github.com/skydive-project/skydive/tree/master/contrib/pipelines/secadvisor/README.md).
+For details, see the instructions found at [Secadvisor Developer's README](https://github.com/skydive-project/skydive/tree/master/contrib/exporters/secadvisor/README.md).
```

Comment:
You have already referenced this README above ... not sure you need to reference it again throughout this blog